modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} agents in a Logical Partition (LPAR)
+
+Use the following procedure to manually add {ibm-z-name} agents to your cluster that runs in an LPAR environment. Use this procedure only for {ibm-z-name} clusters running in an LPAR.
+
+.Prerequisites
+* You have Python 3 installed.
```

Comment:
Rendering issue.

![Screenshot from 2024-08-14 10-10-57](https://github.com/user-attachments/assets/1668652d-00e3-4d30-9475-adda52ecd1a5)

Reply:
+1, looks like you need an extra blank like here:
```suggestion
* You have Python 3 installed.

```